README.md

```diff
@@ -1,5 +1,9 @@
 # Introduction
 
+In order to work properly and properly organize a development project, it is advisable to follow several methods.
+
+Below, we will explain the do's and don'ts to follow in order to carry out a project in good conditions. These are tips, it's up to you to find what works best for you.
+
```

Comment:
We can add a summary to with shortcut to go to differents parts 

see : https://gist.github.com/asabaylus/3071099

Reply:
I added it
Thank you for your review !